test/Internal/PsrMessageStreamTest.php

```diff
@@ -222,10 +224,35 @@ public function testReadReturnsCorrectDataFromStreamReadingTwice(
         string $expectedFirstResult,
         string $expectedSecondResult
     ): void {
-        $inputStream = new PipelineStream(new AsyncGenerator(function () use ($secondChunk, $firstChunk) {
```

Comment:
Wasn't able to find a replacement for PipelineStream - I made it an anonymous class since it appears to only be used for this one test but there's likely a replacement somewhere else; or we could make this its own class.

Reply:
Ah found it - https://github.com/amphp/http-server/pull/325/files#diff-f3a7acd7662e8ff38782ab0e3beba210508103d4ceed70bd363e8856b25d340cL512 - will replace with this in a bit